client/src/pages/SubmissionPage.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import SubmissionForm from '../components/SubmissionForm';
+import Layout from '../components/Layout';
+
+
+function SubmissionPage() {
+    return (
+        <Layout version="hacker">
+            <div>
+                <h1>Submission Page</h1>
+                <SubmissionForm/>
```

Comment:
Great that you made the form its own component. Keep in mind that youll want to pass the submit() function down to this form component as a prop (https://reactjs.org/docs/lifting-state-up.html)